src/averageAges.js

```diff
@@ -20,6 +20,19 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const filteredMen = (century
+    ? people.filter(({ died, sex }) =>
+      Math.ceil(died / 100) === century
+      && sex === 'm')
+    : people.filter(({ sex }) => sex === 'm'));
+
+  const sumOfMenAges = filteredMen.reduce((acc, { died, born }) => {
+    const age = died - born;
+
+    return acc + age;
+  }, 0);
+
+  return sumOfMenAges / filteredMen.length;
```

Comment:
Remember about DRY. You have the same part of logic in the second function. Let's create a function for this